kuronometer-core/src/main/scala/com/github/pedrovgs/kuronometer/free/domain/BuildExecution.scala

```diff
@@ -0,0 +1,37 @@
+package com.github.pedrovgs.kuronometer.free.domain
+
+object BuildTool extends Enumeration {
+  val Gradle, Maven, SBT = Value
+}
+
+object Platform extends Enumeration {
```

Comment:
There is no need for enumarations in Scala unless this is for some compatibility with Java or other JVM langs.
You can have an ADT all with `case object Scala extends Platform`